### DISCUSSION TITLE
How to count elements?

### DISCUSSION THREAD

**Discussion by dynabler** [philippta/flyscrape#44] [Q&A] [answered]
Scenario: you want to count the number of elements on a page, a div.card or table.tr. JavaScipt has a count method: [https://www.geeksforgeeks.org/collect-js-count-function/](https://www.geeksforgeeks.org/collect-js-count-function/)

I can't seem to get it working in Flyscrape. How would you do it?

**Comment by philippta** [comment, marked as answer]
The way you count elements in JavaScript / Flyscrape is:
```javascript
const numberOfCards = doc.find(".card").length;
const numberOfTableRows = doc.find("table tr").length;
```
